src/Http/Features/src/Microsoft.Extensions.Features.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>IFeatureCollection and related components.</Description>
```

Comment:
Nit: The common pattern is to list the commonly used types in this package. See https://www.nuget.org/packages/System.Text.Json/6.0.0-preview.3.21201.4 for e.g.